setup_pip.py

```diff
@@ -109,12 +109,12 @@ def run(self):
 cuda_version = ''.join(os.environ.get('CUDA', 'unknown').split('.')[:2])
 
 install_requires = [
-    'pandas>=0.20,<0.21',
+    'pandas>=0.23.4',
```

Comment:
Why no `<0.24`?

Reply:
> Why no `<0.24`?

The reasons we were actually locked to pandas 0.20.3 were silly little things like a test checking for an exception of "Categorical doesn't ..." and it was updated to "Series doesn't ...". I don't foresee any major reasons we wouldn't be compatible with Pandas 0.24 so it would be nice if Pandas beats us to the next release that it just "works".